docker/bookkeeper/entrypoint.sh

```diff
@@ -8,31 +8,32 @@
 #
 #     http://www.apache.org/licenses/LICENSE-2.0
 #
-set -e
-
-# To create directories for multiple ledgers and journals if specified
-create_dir() {
-  IFS=',' read -ra directories <<< $1
-  for i in "${directories[@]}"
-  do
-      mkdir -p $i
-      if [ "$(id -u)" = '0' ]; then
-          chown -R "${BK_USER}:${BK_USER}" $i
-      fi
-  done
-}
+# set -e
 
 BOOKIE_PORT=${bookiePort:-${BOOKIE_PORT}}
 BOOKIE_PORT=${BOOKIE_PORT:-3181}
+BOOKIE_HTTP_PORT=${BOOKIE_HTTP_PORT:-9090}
 BK_zkServers=$(echo "${ZK_URL:-127.0.0.1:2181}" | sed -r 's/;/,/g')
 ZK_URL=$(echo "${ZK_URL:-127.0.0.1:2181}" | sed -r 's/,/;/g')
 PRAVEGA_PATH=${PRAVEGA_PATH:-"pravega"}
 PRAVEGA_CLUSTER_NAME=${PRAVEGA_CLUSTER_NAME:-"pravega-cluster"}
 BK_CLUSTER_NAME=${BK_CLUSTER_NAME:-"bookkeeper"}
 BK_LEDGERS_PATH="/${PRAVEGA_PATH}/${PRAVEGA_CLUSTER_NAME}/${BK_CLUSTER_NAME}/ledgers"
 BK_DIR="/bk"
+BK_zkLedgersRootPath=${BK_LEDGERS_PATH}
+
+BK_HOME=/opt/bookkeeper
+BINDIR=${BK_HOME}/bin
+BOOKKEEPER=${BINDIR}/bookkeeper
+SCRIPTS_DIR=${BK_HOME}/scripts
 
+export PATH=$PATH:/opt/bookkeeper/bin
+export JAVA_HOME=/usr/lib/jvm/jre-1.8.0
```

Comment:
Are we sure these two paths will be there on the deployment server?

Reply:
Yes, these paths are available in the container as of the v`4.9.2` of the Apache/Bookeeper image we use as the base image.